doc.go

```diff
@@ -77,6 +77,8 @@ Application Options:
       --addcheckpoint=      Add a custom checkpoint.  Format: '<height>:<hash>'
       --nocheckpoints       Disable built-in checkpoints.  Don't do this unless
                             you know what you're doing.
+      --aucomment=          Comments to add to the user agent --
```

Comment:
I also misspelled (plural) it in config.go. All are not --uacomment, like the bitcoind flag. (It's a multi-flag, so can have single comment per flag.)